src/lib/mockserver.js

```diff
@@ -51,7 +51,12 @@ class MockServer {
             await this._dockerSpawnLogs();
         } catch (e) {
             await this.stop(true);
-            throw new Error(`error starting mock container '${this.container}': ${e.message}`);
+            if(e.message){
+                throw new Error(`error starting mock container '${this.container}': ${e.message}`);
+            } 
+            // else
+            console.log(e);
+            throw new Error(`error starting mock container '${this.container}': ${e}`);
```

Comment:
In the flaky tests, `e.message` was undefined. Hence logging the whole error here.
This will help to infirm/confirm the hypothesis that mock-server doesn't start fast enough.

Reply:
The logs go to the `build/test.log` so it wouldn't hurt to always do the full `console.log(e)`, i.e. including stack trace. And then we don't have to repeat the full `e` object in the thrown error (which will format not very readable anyway).

```
 } catch (e) {
    await this.stop(true);
    console.log(e);
    throw new Error(`error starting mock container '${this.container}': ${e.message}`);
}
```

Reply:
`e.message` will be empty if it goes in the catch because of a promise rejection. That's the reason why I changed things a bit here to display the error (which would probably be the rejection string) instead.

Observed behavior is that `e.message` is udnefined when we observe the flakyness and this is logged.

Reply:
Updated to always log the error.